src/Parser/Rogue/Assassination/Modules/Traits/MasterAssassin.js

```diff
@@ -0,0 +1,47 @@
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+
+// Master Assassin: Reduces the cooldown of Vendetta by 10 sec. (Has dimishing returns past rank 3.)
+
+class MasterAssassin extends Analyzer {
+    static dependencies = {
+        combatants: Combatants,
+    };
+
+  on_initialized() {
+      this.active = this.combatants.selected.traitsBySpellId[SPELLS.MASTER_ASSASSIN.id] > 0;
+  }
+
+  vendettaCDReduction = 0;
```

Comment:
This value appears to just be the same as what comes from the getter in `traitCooldownReduction`. It should be removed, and the cdReduction var built in that function should stay local to the function. Also looks like the name was copy pasted 😄